lib/stages/nDiskMultiFormatWriter.hpp

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2022 Kotekan Developers
+
+/****************************************************
+ * @file   nDiskMultiFormatWriter.hpp
+ * @brief  This file declares nDiskMultiFormatWriter
+ *         stage, multi-format multiple drives writer
+ *
+ * @author Mehdi Najafi
+ * @date   12 SEP 2022
+ *****************************************************/
+
+#ifndef N_DISK_MULTIFORMAT_WRITER_HPP
+#define N_DISK_MULTIFORMAT_WRITER_HPP
+
+#include "Config.hpp"          // for Config
+#include "Stage.hpp"           // for Stage
+#include "bufferContainer.hpp" // for bufferContainer
+
+#include <stdint.h> // for uint32_t, int64_t
+#include <string>   // for string
+#include <thread>   // for thread
+#include <time.h>   // for tm
+#include <vector>   // for vector
+
+
+/**
+ * @class nDiskMultiFormatWriter
+ * @brief Consumer ``kotekan::Stage`` which writes input data on multiple drives in various formats.
+ *
+ * This is a consumer which initiates n threads to write to ``n`` disks. Each drive will receive
+ * data from every ``n``th buffer, stored within a common-named subfolder. Within each folder
+ * the data files will be numbered incrementally across the disks.
+ *
+ * @par Buffers
+ *  @buffer in_buf The kotkean buffer with the data to be written
+ *  @buffer_metadata metadata The kotkean metadata to be written
```

Comment:
These comments should indicate the type of frame/metadata the stage expects, in this case it's a `standard` buffer with `any` metadata.

Reply:
Yes, agree. I removed the second one which was about the metadata.